web/src/lib/components/asset-viewer/video-native-viewer.svelte

```diff
@@ -105,43 +66,61 @@
   class="flex h-full select-none place-content-center place-items-center"
   bind:clientWidth={containerWidth}
   bind:clientHeight={containerHeight}
+  use:swipe={() => ({})}
+  onswipe={onSwipe}
 >
-  <video
-    bind:this={videoPlayer}
+  {// vidstack is missing some types for svelte5 event syntax: onauto-play-fail
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  /* @ts-ignore */ void 0}
```

Comment:
Hi, this line is inserting an "undefined" string before `<media-player>`

![image](https://github.com/user-attachments/assets/4be0431f-4e8a-4252-bce1-6dcc4e0f5ae2)

Here's a quick fix
```suggestion
  /* @ts-ignore */ undefined}
```